src/test/java/com/ft/methodearticlemapper/resources/ArticlePreviewTransformationTest.java

```diff
@@ -77,7 +77,7 @@ public void minimalRequirementsMetForContentPreview() {
         UUID expectedUuid = UUID.randomUUID();
         EomFile testEomFile = articlePreviewMinimalEomFile(expectedUuid.toString());
 
-        Content actualContent = postContentToTransformResource.map(testEomFile, IS_PREVIEW, null);
+        Content actualContent = (Content) postContentToTransformResource.map(testEomFile, IS_PREVIEW, null);
```

Comment:
Revert this, there's no need to cast. Check all other occurrences in this test file and revert.